org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/lsp/JavaProtocolExtensions.java

```diff
@@ -66,6 +67,13 @@ public interface JavaProtocolExtensions {
 	@JsonNotification
 	void projectConfigurationUpdate(TextDocumentIdentifier documentUri);
 
+	/**
+	 * Request multiple project configurations update
+	 * @param documentUris the documents from which the project configuration will be updated
+	 */
+	@JsonNotification
+	void projectConfigurationsUpdate(ProjectConfigurationsUpdateParam documentUris);
```

Comment:
I think it makes sense to rename `documentUris` to `params` like the other method declarations in this file.

Then with the javadoc, we can just say it's the parameter object containing the list of documentUris